tests/scalability/replicate-brac-docs/add-docs-to-remote.js

```diff
@@ -0,0 +1,48 @@
+const args = process.argv.slice(2);
+const threadId = args[0];
+const dataDirectory = args[1];
+const config = require('./config.json');
+
+const dataConfig = require('../generate_brac_data/data-config.json');
```

Comment:
I'm uncertain where is file is used. Is this for your local testing?

Reply:
If I'm understanding your question correctly, dataDirectory has to be the base path where all the docs where generated in the generate-brac-data.js script.Maybe what I can do is in that script add that path to the config.json that is also generated.

Reply:
Oh, I'm sorry, I was referring to the whole `add-docs-to-remote.js` file. I don't see it used anywhere, is it?

Reply:
Yes, sorry about that. I didn't finish the jmeter script. Now I added the call there.